src/bindgen.rs

```diff
@@ -36,3 +37,16 @@ pub fn wasm_bindgen_build(path: &str, name: &str) {
         .unwrap_or_else(|e| panic!("{} failed to execute process: {}", emoji::ERROR, e));
     pb.finish();
 }
+
+fn wasm_bindgen_installed() -> bool {
+    if let Ok(path) = env::var("PATH") {
```

Comment:
Normally I'd say we should handle error cases but if your PATH variable isn't set I think you have bigger problems so I think this is fine for now. I plan on going through and cleaning up errors in general later on and I think that is outside the scope of this PR